#! /bin/sh /usr/share/dpatch/dpatch-run
## 60fix_manpage_name_section.dpatch by Manuel Prinz <debian@pinguinkiste.de>
##
## DP: Fixes lintian warnings "manpage-has-bad-whatis-entry".

@DPATCH@
diff -urNad openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Comm_f2c.3 openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Comm_f2c.3
--- openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Comm_f2c.3	2007-02-16 19:20:45.000000000 +0100
+++ openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Comm_f2c.3	2007-12-14 13:13:04.000000000 +0100
@@ -2,7 +2,7 @@
 .\" Copyright (c) 1996 Thinking Machines Corporation
 .TH MPI_Comm_f2c 3OpenMPI "September 2006" "Open MPI 1.2" " "
 .SH NAME
-\fBHandle translation routines \fP \- Translates a C handle into a Fortran handle, or vice versa. 
+\fBMPI_Comm_f2c \fP \- Translates a C handle into a Fortran handle, or vice versa.
 
 .SH SYNTAX
 .ft R
diff -urNad openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Status_f2c.3 openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Status_f2c.3
--- openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Status_f2c.3	2007-02-16 19:20:45.000000000 +0100
+++ openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Status_f2c.3	2007-12-14 13:13:04.000000000 +0100
@@ -2,7 +2,7 @@
 .\" Copyright (c) 1996 Thinking Machines Corporation
 .TH MPI_Status_f2c 3OpenMPI "September 2006" "Open MPI 1.2" " "
 .SH NAME
-\fBStatus conversion routines \fP \- Translates a C status into a Fortran status, or vice versa. 
+\fBMPI_Status_f2c \fP \- Translates a C status into a Fortran status, or vice versa.
 
 .SH SYNTAX
 .ft R
diff -urNad openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Type_create_darray.3 openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Type_create_darray.3
--- openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Type_create_darray.3	2007-02-16 19:20:45.000000000 +0100
+++ openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Type_create_darray.3	2007-12-14 13:13:04.000000000 +0100
@@ -2,7 +2,7 @@
 .\"Copyright (c) 1996 Thinking Machines
 .TH MPI_Type_create_darray 3OpenMPI "September 2006" "Open MPI 1.2" " "
 .SH NAME
-\fBMPI_Type_create_darray\fP \- 
+\fBMPI_Type_create_darray\fP \- Creates a darray.
 
 .SH SYNTAX
 .ft R
diff -urNad openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_create_errhandler.3 openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_create_errhandler.3
--- openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_create_errhandler.3	2007-02-16 19:20:45.000000000 +0100
+++ openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_create_errhandler.3	2007-12-14 13:13:04.000000000 +0100
@@ -2,7 +2,7 @@
 .\" Copyright (c) 1996 Thinking Machines Corporation
 .TH MPI_Win_create_errhandler 3OpenMPI "September 2006" "Open MPI 1.2" " "
 .SH NAME
-\fBMPI_Win_create_errhandler\fP \- 
+\fBMPI_Win_create_errhandler\fP \- Creates an error handler for a window.
 
 .SH SYNTAX
 .ft R
diff -urNad openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_create_keyval.3 openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_create_keyval.3
--- openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_create_keyval.3	2007-02-16 19:20:45.000000000 +0100
+++ openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_create_keyval.3	2007-12-14 13:13:04.000000000 +0100
@@ -2,7 +2,7 @@
 .\" Copyright (c) 1996 Thinking Machines Corporation
 .TH MPI_Win_create_keyval 3OpenMPI "September 2006" "Open MPI 1.2" " "
 .SH NAME
-\fBMPI_Win_create_keyval\fP \- 
+\fBMPI_Win_create_keyval\fP \- Creates a key value.
 
 .SH SYNTAX
 .ft R
diff -urNad openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_delete_attr.3 openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_delete_attr.3
--- openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_delete_attr.3	2007-02-16 19:20:45.000000000 +0100
+++ openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_delete_attr.3	2007-12-14 13:13:04.000000000 +0100
@@ -2,7 +2,7 @@
 .\" Copyright (c) 1996 Thinking Machines Corporation
 .TH MPI_Win_delete_attr 3OpenMPI "September 2006" "Open MPI 1.2" " "
 .SH NAME
-\fBMPI_Win_delete_attr\fP \- 
+\fBMPI_Win_delete_attr\fP \- Deletes an attribute.
 
 .SH SYNTAX
 .ft R
diff -urNad openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_free_keyval.3 openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_free_keyval.3
--- openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_free_keyval.3	2007-02-16 19:20:45.000000000 +0100
+++ openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_free_keyval.3	2007-12-14 13:13:04.000000000 +0100
@@ -2,7 +2,7 @@
 .\" Copyright (c) 1996 Thinking Machines Corporation
 .TH MPI_Win_free_keyval 3OpenMPI "September 2006" "Open MPI 1.2" " "
 .SH NAME
-\fBMPI_Win_free_keyval\fP \- 
+\fBMPI_Win_free_keyval\fP \- Frees a key value.
 
 .SH SYNTAX
 .ft R
diff -urNad openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_get_name.3 openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_get_name.3
--- openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_get_name.3	2007-02-16 19:20:45.000000000 +0100
+++ openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_get_name.3	2007-12-14 13:13:04.000000000 +0100
@@ -2,7 +2,7 @@
 .\" Copyright (c) 1996 Thinking Machines Corporation
 .TH MPI_Win_get_name 3OpenMPI "September 2006" "Open MPI 1.2" " "
 .SH NAME
-\fBMPI_Win_get_name\fP \- 
+\fBMPI_Win_get_name\fP \- Obtains the name of a window.
 
 .SH SYNTAX
 .ft R
diff -urNad openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_set_attr.3 openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_set_attr.3
--- openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_set_attr.3	2007-02-16 19:20:45.000000000 +0100
+++ openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_set_attr.3	2007-12-14 13:13:04.000000000 +0100
@@ -2,7 +2,7 @@
 .\" Copyright (c) 1996 Thinking Machines Corporation
 .TH MPI_Win_set_attr 3OpenMPI "September 2006" "Open MPI 1.2" " "
 .SH NAME
-\fBMPI_Win_set_attr\fP \- 
+\fBMPI_Win_set_attr\fP \- Sets the value of a window attribute.
 
 .SH SYNTAX
 .ft R
diff -urNad openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_set_name.3 openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_set_name.3
--- openmpi-1.2.4.orig~/ompi/mpi/man/man3/MPI_Win_set_name.3	2007-02-16 19:20:45.000000000 +0100
+++ openmpi-1.2.4.orig/ompi/mpi/man/man3/MPI_Win_set_name.3	2007-12-14 13:13:04.000000000 +0100
@@ -2,7 +2,7 @@
 .\" Copyright (c) 1996 Thinking Machines Corporation
 .TH MPI_Win_set_name 3OpenMPI "September 2006" "Open MPI 1.2" " "
 .SH NAME
-\fBMPI_Win_set_name\fP \- 
+\fBMPI_Win_set_name\fP \- Sets the name of a window.
 
 .SH SYNTAX
 .ft R
